src/components/widgets/LineChart.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  LineChart,
+  Line,
+  CartesianGrid,
+  XAxis,
+  YAxis,
+  Tooltip,
+} from 'recharts';
+
+const CustomLineChart = (props) => {
+  const { ...otherProps } = props;
+  return (
+    <LineChart
+      data={otherProps.data}
+      height={otherProps.height}
+      margin={{ top: 5, right: 30, left: 20, bottom: 5 }}
```

Comment:
@rhenshaw56 I'm not sure what these numbers in margin do, but we need to make this component as customizable as possible. so if another project needs to use this component, changing the values in margin should not affect our app, what do you think?

Reply:
You can do like `margin={
{ top: {otherProps.top || 5}, 
right: {otherProps.right || 30}
}`